apps/rewards/app/components/Content/MyRewards.js

```diff
@@ -14,48 +16,59 @@ import {
   ONE_TIME_DIVIDEND,
   RECURRING_DIVIDEND,
   ONE_TIME_MERIT,
+  READY,
+  CLAIMED,
 } from '../../utils/constants'
 import { Empty } from '../Card'
 import Metrics from './Metrics'
-import { useAppState } from '@aragon/api-react'
+import { useAppState, useNetwork } from '@aragon/api-react'
 import BigNumber from 'bignumber.js'
-import { displayCurrency, getStatus } from '../../utils/helpers'
+import { displayCurrency, getStatus, getStatusId } from '../../utils/helpers'
 
 const MyRewards = ({
   myRewards,
   myMetrics,
   viewReward,
   claimReward,
+  claimHashes,
 }) => {
+  const network = useNetwork()
   const rewardsEmpty = myRewards.length === 0
 
+  const getEtherscanLink = reward => {
+    const networkChunk = network.id === 1 ? '' : `${network.type}.`
```

Comment:
I could have used `network.type`, but I'm not sure about it's return value for mainnet. If it's `mainnet`, it's not useful because `mainnet.etherscan.io` doesn't work. The documentation is rather laconic about it, and I couldn't find code examples. In any case, checking the `id` seems more robust. Obviously, this hasn't been tested.

Reply:
Can you test this and check? Does it use the user-configured setting from MetaMask?

Reply:
What I meant by this not having been tested is that it can only be tested on mainnet. Or am I missing something? What setting from Metamask? If you're wondering about the network id, there's no doubt that mainnet is 1.

Reply:
I guess I'm wondering if `useNetwork()` responds to the user setting in MetaMask. If it does, then you can set MM to mainnet and then check `network.type` to see what it looks like. 

How does `useNetwork()` check what network its on, other than using MetaMask? This might be a question for @topocount or @Quazia.

Reply:
That's just an interface that gets an updated value for the Aragon's api `network` observable: 
https://github.com/aragon/aragon.js/blob/e4a155ef82420147d4926756861a5bd6cd46f84f/packages/aragon-api/src/index.js#L79

It gets the value from the current provider, it can be MM or other tool injecting the provider.